.github/workflows/continuous-integration.yml

```diff
@@ -5,9 +5,6 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
-    paths-ignore:
```

Comment:
question

why did you remove this?

Reply:
Because as we discovered in a different branch if we actually trigger this then CI won't run (as you would expect). The issue is all of our checks for this workflow are marked as required status checks. You can't merge the branch without the status check passing which will never be generated because the workflow never runs.